airflow/contrib/hooks/gcs_hook.py

```diff
@@ -531,7 +535,7 @@ def compose(self, bucket_name, source_objects, destination_object):
         :type destination_object: str
         """
 
-        if not source_objects or not len(source_objects):
+        if not source_objects or not source_objects:
```

Comment:
It looks strange. Does it work properly?

Reply:
Yes, it is recommended way of checking if list has any elements. 
It works because `bool([]) == False)`.